workflow/rules/purecn.smk

```diff
@@ -0,0 +1,112 @@
+__author__ = "Erik Demitz-Helin"
+__copyright__ = "Copyright 2022, Erik Demitz-Helin"
+__email__ = "erik.demitz-helin@gu.se"
+__license__ = "GPL-3"
+
+
+rule purecn_coverage:
+    input:
+        bam="alignment/samtools_merge_bam/{sample}_{type}.bam",
+        bai="alignment/samtools_merge_bam/{sample}_{type}.bam.bai",
+    output:
+        temp(
+            expand(
+                "cnv_sv/purecn_coverage/{{sample}}_{{type}}{ext}",
+                ext=[
+                    "_coverage.txt.gz",
+                    "_coverage_loess.txt.gz",
+                    "_coverage_loess.png",
+                    "_coverage_loess_qc.txt",
+                ]
+            )
+        ),
+    params:
+        intervals=config.get("purecn_coverage", {}).get("intervals", ""),
+        extra=config.get("purecn_coverage", {}).get("extra", ""),
+    log:
+        "cnv_sv/purecn_coverage/{sample}_{type}.output.log",
+    benchmark:
+        repeat(
+            "cnv_sv/purecn_coverage/{sample}_{type}.output.benchmark.tsv",
+            config.get("purecn_coverage", {}).get("benchmark_repeats", 1),
+        )
+    threads: config.get("purecn_coverage", {}).get("threads", config["default_resources"]["threads"])
+    resources:
+        mem_mb=config.get("purecn_coverage", {}).get("mem_mb", config["default_resources"]["mem_mb"]),
+        mem_per_cpu=config.get("purecn_coverage", {}).get("mem_per_cpu", config["default_resources"]["mem_per_cpu"]),
+        partition=config.get("purecn_coverage", {}).get("partition", config["default_resources"]["partition"]),
+        threads=config.get("purecn_coverage", {}).get("threads", config["default_resources"]["threads"]),
+        time=config.get("purecn_coverage", {}).get("time", config["default_resources"]["time"]),
+    container:
+        config.get("purecn_coverage", {}).get("container", config["default_container"])
+    conda:
+        "../envs/purecn_coverage.yaml"
+    message:
+        "{rule}: Calculate coverage for {wildcards.sample}"
+    shell:
+        "(Rscript $PURECN/Coverage.R "
+        "--out-dir=cnv_sv/purecn_coverage "
+        "--bam={input.bam} "
+        "--intervals={params.intervals} "
+        "--cores={threads} "
+        "{params.extra}) &> {log}"
+
+
+rule purecn:
+    input:
+        unpack(get_purecn_inputs),
+        vcf="snv_indels/gatk_mutect2/{sample}_{type}.merged.softfiltered.vcf.gz",
+        tbi="snv_indels/gatk_mutect2/{sample}_{type}.merged.softfiltered.vcf.gz.tbi",
```

Comment:
```suggestion
        vcf="snv_indels/bcbio_variation_recall_ensemble/{sample}_{type}.ensembled.vcf.gz",
        tbi="snv_indels/bcbio_variation_recall_ensemble/{sample}_{type}.ensembled.vcf.gz.tbi",
```

Reply:
This currently will not work as PureCN crashes when the way BQ is reported is not consistent between callers (lima1/PureCN#249)